lib/renogen/cli.rb

```diff
@@ -67,6 +67,7 @@ def self.init
     #
     # @param ticket_name [String]
     def self.new_ticket(ticket_name)
+      raise 'You need to provide a ticket_name' if ticket_name.nil?
```

Comment:
I think its ok here because without it. It will create a file with only the .yml extension.